### DISCUSSION TITLE
Could you also provide Qwerty-FR for the ANSI layout ?

### DISCUSSION THREAD

**Discussion by Psebcool** [qwerty-fr/qwerty-fr#78] [Ideas]
I love your Qwerty-FR ISO, it's really helpful and better than standard ISO FR layout. 

It would be great if you could also provide the qwerty-fr in the ANSI layout.
Because it is often easier to find ANSI (US) than ISO (FR/UK/DE) keyboards from certain suppliers. And also because the ANSI (US) layout has better accuracy about the inscriptions on the keycaps with your Qwerty-FR mod :)

Thanks for all !

**Comment by devnoname120** [comment]
The QWERTY-fr is based on [US ANSI/ISO (terminal style)](https://www.farah.cl/Keyboardery/A-Visual-Comparison-of-Different-National-Layouts/#enUS).

What issues do you have when you use the keyboard layout with an ANSI keyboard? It should work without problems.

**Reply by Psebcool** [reply]
Oh sorry ! I didn't know that Qwerty-fr was also compatible with ANSI. Because on the readme, I see only the ISO layout picture. I plan to buy an ANSI keyboard soon, it's for the reason I asked.

Thanks !

**Comment by Psebcool** [comment]
I did a ANSI layout picture for QWERTY-fr :

![ANSI keyboard with QWERTY-FR layout (by Pseb)](https://github.com/user-attachments/assets/b7211d23-1965-4edf-bd27-744a6bccfe70)

You can reuse it for the github page if you want.

**Reply by devnoname120** [reply]
This would resolve the following issue: https://github.com/qwerty-fr/qwerty-fr/issues/79

Note that some keys seem to be missing. For example:
- The circumflex `ˆ` dead key on <kbd>Option ⌥</kbd> <kbd>6</kbd>
- The integral `∫` dead key for the math mode on <kbd>Option ⌥</kbd> <kbd>m</kbd>
- The grave accent `` ` `` dead key on <kbd>Option ⌥</kbd> <kbd>`</kbd>
- The tilde `˜` dead key on <kbd>Shift ⇧</kbd> <kbd>Option ⌥</kbd> <kbd>`</kbd>
- Etc.

Could you fix these issues and provide the PSD/vector/other of the image? Unless you designed it exclusively using PNG.

Thank you, it's much appreciated!

**Reply by Psebcool** [reply]
@devnoname120 

Of course I will send you the PSD on your github mailbox.

For the keys missing, they not work in my side I don't know why. I use ANSI international keyboard with qwerty-fr installed. So I'll let you add them via the PSD.

;)
